### PR TITLE
feat(settings): Show supported apps on app settings if subscribed

### PR DESCRIPTION
**Pull request by susnux**
## Screenshots :framed_picture: 

image

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [x] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [x] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Review by st3iny — APPROVED** [review]
Epic! Tested and works.

File: apps/settings/src/views/Apps.vue [hunk 1]
```diff
@@ -255,6 +265,9 @@ export default {
 		changelog() {
 			return (release) => release.translations.en.changelog
 		},
+		isSubscribed() {
+			return this.apps.some(app => app.level === 300)
```
Comment (szaimen) [inline apps/settings/src/views/Apps.vue:269]: Can you add a comment here what app level 300 means?

File: apps/settings/src/components/AppList.vue [hunk 2]
```diff
@@ -154,6 +154,9 @@ export default {
 			if (this.category === 'updates') {
 				return apps.filter(app => app.update)
 			}
+			if (this.category === 'supported') {
+				return apps.filter(app => app.level === 300)
```
Comment (szaimen) [inline apps/settings/src/components/AppList.vue:158]: here as well?

**Review by szaimen — APPROVED** [review]
LGTM but didnt test

**Review by AndyScherzinger — APPROVED** [review]

**Review by skjnldsv — APPROVED** [review]

**Pull request merged**